sdv/single_table/ctgan.py

```diff
@@ -71,15 +71,17 @@ def get_loss_values_plot(self, title='CTGAN loss function'):
 
         # Tidy up the loss values data
         loss_df = self._model.loss_values.copy()
-        loss_df['Generator Loss'] = loss_df['Generator Loss'].apply(lambda x: x.item())
-        loss_df['Discriminator Loss'] = loss_df['Discriminator Loss'].apply(lambda x: x.item())
+        loss_df['Generator Loss'] = loss_df['Generator Loss'].apply(
+            lambda x: x.item() if isinstance(x, pd.Series) else x)
```

Comment:
Is it ever not a series?

Reply:
Yeah I believed something changed where each element had an `item()` method associated to it because it was a `pd.Series` ([example](https://github.com/sdv-dev/SDV/issues/1828)) now the elements within it are now just floats.